src/tests/ngraph_helpers/snippets_ngraph_functions/include/subgraph_customizable.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/ngraph.hpp"
+#include "snippets_helpers.hpp"
+
+/* This file contains definitions of rather complex functions (models) that support (and require)
+ * specification of some the internal operations. This flexibility is required to extend coverage of
+ * different tokenization scenarios in parametrized tests. All the functions are expected to be direct
+ * descendants of SnippetsFunctionCustomizable (defined in snippets_helpers.hpp).
+ */
+
+namespace ov {
+namespace test {
+namespace snippets {
+/// Convolution followed by a two-input Multiply, Relu and Sqrt
+/// Tokenized by attaching eltwises, but becomes non-tokenizable if Multiply is substituted with Add (CPU-specific fusing)
+//    in1          in2
+// Convolution   Convert
+//         Multiply
+//           Relu
+//           Sqrt
+//          Result
+class ConvMulActivation : public SnippetsFunctionCustomizable {
```

Comment:
Renamed, thanks for noticing.

Reply:
Regarding https://github.com/openvinotoolkit/openvino/pull/10605#discussion_r830525287 :
I think we shouldn't do it, because the lowering will be performed in SetUp in this case. We however need to test the lowering, so it should be performed inside TEST_P instead. Please see also thi reply https://github.com/openvinotoolkit/openvino/pull/10605#discussion_r830815416